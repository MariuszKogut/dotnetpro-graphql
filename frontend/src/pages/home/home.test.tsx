import React from 'react'
import { render } from '@testing-library/react'
import Home from './home'

jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    history: jest.fn(),
  }),
}))

describe('Home', () => {
  it('renders without errors', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
