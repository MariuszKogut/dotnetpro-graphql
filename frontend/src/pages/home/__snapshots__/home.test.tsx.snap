// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without errors 1`] = `
<div>
  <div
    class="jumbotron"
  >
    <h1>
      Willkommen!
    </h1>
    <p
      class="lead"
    >
      Sch√∂n, dass Sie customer-app aufgerufen haben.
    </p>
    <hr
      class="my-4"
    />
    <p>
      Verwalten Sie Ihre Kunden wie noch nie zuvor. Dank React im Frontend und .NET Core im Backend.
    </p>
    <p
      class="lead"
    >
      <button
        class="btn btn-primary btn-lg"
        type="button"
      >
        Zur Kundenliste
      </button>
    </p>
  </div>
</div>
`;
